utils.py

```diff
@@ -12,11 +12,11 @@
 
 
 def get_ckpt(path):
-	ckpt=path 
+	ckpt=path
 	ckpt = torch.load(ckpt, map_location='cpu')
 	plain_ckpt={}
 	for k in ckpt.keys():
-		plain_ckpt[k[7:]] = ckpt[k] # torch DDP models have a extra wrapper of module., so to remove that... 
+		plain_ckpt[k[7:]] = ckpt[k] # remove the 'module' portion of key if model is Pytorch DDP
```

Comment:
You can use soft load I think. Check "soft" in torch.load.

Reply:
I wasn't able to find this in torch documentation for `load` or `load_state_dict`. Should this current implementation be made more clear?

Reply:
I am unable to find, but it should be OK.